manim/mobject/coordinate_systems.py

```diff
@@ -699,6 +706,462 @@ def prepare_for_nonlinear_transform(self, num_inserted_curves=50):
         return self
 
 
+class PolarPlane(Axes):
+    r"""Creates a polar plane with background lines.
+
+    Parameters
+    ----------
+    azimuth_step
+        The number of divisions in the azimuth. If ``None`` is specified then it will use the default
+        specified by ``azimuth_units``:
+
+        - ``"PI radians"`` or ``"TAU radians"``: 20
+        - ``"degrees"``: 36
+        - ``"gradians"``: 40
+        - ``None``: 1
+
+        A non-integer value will result in a partial division at the end of the circle.
+
+    size
+        The diameter of the plane.
+
+    radius_step
+        The step of faded lines in the radius.
+
+    radius_max
+        The maximum value of the radius.
+
+    azimuth_units
+        Specifies a default labelling system for the azimuth. Choices are:
+
+        - ``"PI radians"``: Fractional labels in the interval :math:`\left[0, 2\pi\right]` with :math:`\pi` as a constant.
+        - ``"TAU radians"``: Fractional labels in the interval :math:`\left[0, \tau\right]` (where :math:`\tau = 2\pi`) with :math:`\tau` as a constant.
+        - ``"degrees"``: Decimal labels in the interval :math:`\left[0, 360\right]` with a degree (:math:`^{\circ}`) symbol.
+        - ``"gradians"``: Decimal lables in the interval :math:`\left[0, 400\right]` with a superscript "g" (:math:`^{g}`).
+        - ``None``: Decimal labels in the interval :math:`\left[0, 1\right]`.
+
+    azimuth_offset
+        The angle offset of the azimuth, expressed in radians.
+
+    azimuth_direction
+        The direction of the azimuth.
+
+        - ``"CW"``: Clockwise.
+        - ``"CCW"``: Anti-clockwise.
+
+    azimuth_label_buff
+        The buffer for the azimuth labels.
+
+    azimuth_label_scale
+        The scale of the azimuth labels.
+
+    radius_config
+        The axis config for the radius.
+
+    Examples
+    --------
+
+    .. manim:: PolarPlaneExample
```

Comment:
Two suggestions, definitely non-blocking:
- I would like the labels to be a bit bigger here.
- I would like the option to have the pi / tau after the fraction - but only add that if you like :)

Reply:
I've scaled them up just for that example. As for the option, I've added `azimuth_compact_fraction` as a boolean, default `True`. When `False` it does what you asked for, see below:
![A_ManimCE_v0 6 0](https://user-images.githubusercontent.com/25301457/118699815-e4eebe00-b809-11eb-8ac2-98fba0220f45.png)

Reply:
Very nice! Thank you :)